features/plugins.feature

```diff
@@ -1,37 +1,20 @@
 Feature: Plugins Work
 
+  Background:
+    Given a mocked home directory
+
   @slow_process @unstable
   Scenario: Lolcommits.com integration works
-    Given I am in a git repository named "dot_com" with lolcommits enabled
-    When I run `lolcommits --config` and wait for output
-    And I enter "dot_com" for "Plugin Name"
-    And I enter "true" for "enabled"
-    And I enter "b2a70ac0b64e012fa61522000a8c42dc" for "api_key"
-    And I enter "b2a720b0b64e012fa61522000a8c42dc" for "api_secret"
-    And I enter "c4aed530b64e012fa61522000a8c42dc" for "repo_id"
-    Then I should be presented "Successfully configured plugin: dot_com"
-    When I do a git commit
-    Then the output should contain "*** Preserving this moment in history."
-    And there should be exactly 1 jpg in "../.lolcommits/dot_com"
-
-  Scenario: Disable loltext
-    Given I am in a git repository named "loltext" with lolcommits enabled
-    And I run `lolcommits --config` and wait for output
-    And I enter "loltext" for "Plugin Name"
-    And I enter "false" for "enabled"
-    Then I should be presented "Successfully configured plugin: loltext"
-    When I do a git commit
-    Then the output should contain "*** Preserving this moment in history."
-    And there should be exactly 1 jpg in "../.lolcommits/loltext"
-
-  Scenario: lolsrv integration works
-    Given I am in a git repository named "lolsrv" with lolcommits enabled
-    When I run `lolcommits --config` and wait for output
-    And I enter "lolsrv" for "Plugin Name"
-    And I enter "true" for "enabled"
-    And I enter "http://localhost" for "server"
-    Then I should be presented "Successfully configured plugin: lolsrv"
-    When I do a git commit
-    Then the output should contain "*** Preserving this moment in history."
-    And there should be exactly 1 jpg in "../.lolcommits/lolsrv"
-
+    Given I am in a git repo named "dot_com" with lolcommits enabled
+    When I run `lolcommits --config` interactively
+      And I wait for output to contain "Name of plugin to configure:"
+      Then I type "dot_com"
+      And I wait for output to contain "enabled:"
+      Then I type "true"
+      And I wait for output to contain "api_key:"
+      Then I type "b2a70ac0b64e012fa61522000a8c42dc"
+      And I wait for output to contain "api_secret:"
+      Then I type "b2a70ac0b64e012fa61522000a8c42dc"
+      And I wait for output to contain "repo_id:"
+      Then I type "b2a70ac0b64e012fa61522000a8c42dc"
+    Then the output should contain "Successfully configured plugin: dot_com"
```

Comment:
Not a fan of tests like this one, it does test that the UI to configure the plugin works, but doesn't test that the config file actually contains those values, or that the plugin actually submits the lolcommit on post captures.  Its no worse that what we already have, so its fine for now.

I guess when we move these to their own gems, they'll have their own tests their.